remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RulesImpl.java

```diff
@@ -112,6 +119,39 @@ public void deleteRuleById(String folderNodeId, String ruleSetId, String ruleId)
         ruleService.removeRule(folderNodeRef, rule);
     }
 
+    @Override
+    public RuleSetLink linkToRuleSet(String folderNodeId, String linkToNodeId) {
+
+        RuleSetLink ruleSetLink = new RuleSetLink();
+        final NodeRef folderNodeRef = validator.validateFolderNode(folderNodeId,false);
+        final NodeRef linkToNodeRef = validator.validateFolderNode(linkToNodeId, false);
+
+        if(nodeService.exists(folderNodeRef))
```

Comment:
What should happen if the folderNodeRef doesn't exist?  Is this already validated above?